packages/browser-wallet/src/popup/shared/utils/seedPhrase-helpers.ts

```diff
@@ -3,7 +3,7 @@ import { useAtomValue } from 'jotai';
 import { encryptedSeedPhraseAtom, sessionPasscodeAtom } from '@popup/store/settings';
 import { noOp, useAsyncMemo } from 'wallet-common-helpers';
 import { mnemonicToSeedSync } from '@scure/bip39';
-import { decrypt } from '../crypto';
+import { decrypt } from '@shared/utils/crypto';
```

Comment:
If you're fixing file placements, then you might as well also re-name this file to:
```
seed-phrase-helpers.ts
```